### DISCUSSION TITLE
Support s3 bulket as channel

### DISCUSSION THREAD

**Discussion by fecet** [prefix-dev/pixi#2125] [General]
Since we already support using files as custom channels, using S3 as a channel shouldn't be difficult either. This would provide convenience for many enterprise use cases and also reduce the load on the prefix server.

**Comment by wolfv** [comment]
Hi @fecet - yes, we would be happy to merge a PR to that effect in `rattler`. We already support the `gcs://` and `oci://` prefixes and could similarly support an `s3://` prefix. We have a middleware that could do the necessary request signing. Just someone needs to implement the bits and pieces :) Probably there is a crate to do it, too!
